libz-rs-sys/src/gz.rs

```diff
@@ -101,15 +101,17 @@ pub unsafe extern "C-unwind" fn gzopen(path: *const c_char, mode: *const c_char)
 ///
 /// # Safety
 /// The caller must ensure that path and mode are NULL or point to valid C strings.
-unsafe fn gzopen_help(path: *const c_char, fd: c_int, mode: *const c_char) -> gzFile {
+fn gzopen_help(path: *const c_char, fd: c_int, mode: *const c_char) -> gzFile {
```

Comment:
this function is still unsafe: it has preconditions that the caller must satisfy, that are not verified by the compiler.